utils.go

```diff
@@ -60,9 +60,12 @@ func FindFilterFunction(funcName string) (string, error) {
 
 	// Cache available filter functions if necessary
 	if len(availableFilterFunctions) == 0 {
-		funcs, err := os.ReadFile("/sys/kernel/debug/tracing/available_filter_functions")
+		funcs, err := os.ReadFile("/sys/kernel/tracing/available_filter_functions")
 		if err != nil {
```

Comment:
Do we want to fallback and check `/sys/kernel/debug/tracing` in all error cases? Perhaps we should only fallback if `errors.Is(err, fs.ErrNotExist)`? I'm not 100% sure on what the possible errors could be, if this isn't mounted or mounted improperly.

I'm looking at the scenario where `/sys/kernel/tracing` is correct, but is failing because of some other misconfiguration. The code as is would report an error about `/sys/kernel/debug/tracing` not existing (if it doesn't), rather than the potential real error about `/sys/kernel/tracing`.

Reply:
Yeah I'm not sure neither. I've applied your suggestion to fallback only on ErrNotExist. Thank you